### PR TITLE
Pimp My Terminal

### PR DESCRIPTION
**Pull request by dyc3**

### CODE REVIEW

File: sec.club/src/views/PimpMyTerminal/PimpMyTerminal.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+			width: [300, 400, 300],
+			scaleY: [1, 0.9, 1],
+		},
+		transition: {
+			duration: 0.6,
+			delay: 1.5,
+			ease: "easeInOut",
+			times: [0, 0.5, 1],
+		},
+	};
```
Comment (carpenterd777) [inline sec.club/src/views/PimpMyTerminal/PimpMyTerminal.js]: I like these animations!

File: sec.club/src/articles/events/pimp-my-terminal/pimp-my-terminal.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+(especially on windows) it probably sucks.
+
+### Windows
+
+Windows users should take a look at the [new Windows terminal from Microsoft](https://github.com/microsoft/terminal).
+You can install it from the Windows Store.
+
+There are other options:
+
+- [Windows Terminal](https://github.com/microsoft/terminal)
```
Comment (carpenterd777) [inline sec.club/src/articles/events/pimp-my-terminal/pimp-my-terminal.md]: I think its a little confusing to list Windows Terminal as another option when it was just discussed above

Comment (dyc3) [reply sec.club/src/articles/events/pimp-my-terminal/pimp-my-terminal.md]: My thinking was that if somebody was just glossing over the content and not actually reading it, they wouldn't see windows terminal listed above

File: sec.club/src/articles/events/pimp-my-terminal/pimp-my-terminal.md [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+- [kitty](https://sw.kovidgoyal.net/kitty/)
+- [Cool Retro Term](https://github.com/Swordfish90/cool-retro-term) - Very fancy, not battery friendly
+- [st](https://st.suckless.org)
+
+### Mac
+
+Most Linux terminals should work on Mac, but your milage may vary.
+
+- [iTerm2](https://www.iterm2.com)
+
```
Comment (carpenterd777) [inline sec.club/src/articles/events/pimp-my-terminal/pimp-my-terminal.md]: Hyper is fully supported for Mac and Windows, consider adding it to those lists too?

Comment (dyc3) [reply sec.club/src/articles/events/pimp-my-terminal/pimp-my-terminal.md]: Good catch

File: sec.club/src/articles/events/pimp-my-terminal/pimp-my-terminal.md (hunk 2 above)
Comment (carpenterd777) [reply sec.club/src/articles/events/pimp-my-terminal/pimp-my-terminal.md]: Fair enough

**Pull request merged**